### ISSUE TITLE
Parallel support for GNC projects

### ISSUE THREAD

**Issue by AndreasRIHU** [MODFLOW-USGS/modflow6#1319]
Hi again,
are projects containing GNC packages currently supported in parallel mode?
I've tried several models of that type and always get an error similiar to that one:

  ERROR REPORT:
  1. Column number in list (16) is outside of the grid. Cell number cannot be
     determined for cellid (1 1 16) and results in a user node number (-1)
     that is outside of the grid.
  UNIT ERROR REPORT:
  1. Error occurred while reading file 'simulation.gnc'
  
Besides: have you got any document decribing the current restrictions in parallel mode in order to avoid opening issues on every one of them?

Thanks
Andreas

**Comment by AndreasRIHU** [comment]
Hi Chris,
understood. I'm probably going to postpone further testing till fall, when more documentation regarding the parallel part will be available and there will still be time for discussions :-)
Thanks
Andreas
